api_app/connectors_manager/connectors/yeti.py

```diff
@@ -0,0 +1,81 @@
+# This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
+# See the file 'LICENSE' for copying permission.
+
+from api_app.exceptions import ConnectorRunException
+from django.conf import settings
+
+import pyeti
```

Comment:
nitpick: move this import to top.

Reply:
and keep the `api_app` imports together.

Reply:
sometimes I forget to check auto imports :sweat_smile: